### PR TITLE
Add credentials v2 context & multikey context to documentLoader.

### PR DESCRIPTION
**Pull request by aljones15**
Adds credentials V2 and multikey contexts to the documentLoader.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
 ### Fixed
 - Revert to `node:20-alpine` in Dockerfile.
 
+### Added
```
Comment (dlongley) [inline CHANGELOG.md:5]: Alphabetize (move above `Fixed`).

File: lib/documentLoader.js [hunk 2]
```diff
@@ -1,12 +1,23 @@
 /*!
  * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
```
Comment (dlongley) [inline lib/documentLoader.js]:
```suggestion
 * Copyright (c) 2023-2024 Digital Bazaar, Inc. All rights reserved.
```

File: lib/documentLoader.js [hunk 3]
```diff
@@ -1,12 +1,23 @@
...
 import {securityLoader} from '@digitalbazaar/security-document-loader';
 
 export const loader = securityLoader();
 
-loader.addStatic(
-  dataIntegrityContext.constants.CONTEXT_URL,
-  dataIntegrityContext.contexts.get(
-    dataIntegrityContext.constants.CONTEXT_URL));
+const addContext = ({url, contexts}) =>
+  loader.addStatic(url, contexts.get(url));
```
Comment (dlongley) [inline lib/documentLoader.js]: Is this `addContext` helper supposed to enumerate `contexts` or something? It doesn't seem to do anything more than what `loader.addStatic` is already doing.

Is there more than one context from the DI context module that should be getting added here?

**Review by dlongley — CHANGES_REQUESTED** [review]

File: lib/documentLoader.js (hunk 3 above)
Comment (aljones15) [reply lib/documentLoader.js]: just made the code look cleaner as you did have to fetch a context from a Map with a really long name. I think it improves readability.

File: CHANGELOG.md (hunk 1 above)
Comment (aljones15) [reply CHANGELOG.md:5]: alphabetized here: https://github.com/digitalbazaar/vc-holder-http/pull/4/commits/142c117f25d479aeecc0d099ba482e899e598e55

File: lib/documentLoader.js (hunk 3 above)
Comment (aljones15) [reply lib/documentLoader.js]: changed from `addContext` to `addContexts` here: https://github.com/digitalbazaar/vc-holder-http/pull/4/commits/5c99d73b55ce9122e3fe4cfffe11db8915258944

It now adds all values in the contexts map.

**Review by dlongley — APPROVED** [review]

**Pull request merged**